roles/openshift_logging/tasks/install_elasticsearch.yaml

```diff
@@ -134,6 +136,8 @@
     openshift_logging_es_recover_after_time: "{{openshift_logging_es_ops_recover_after_time}}"
```

Comment:
Per my IRC comment, the difference is when running the 'ops' tasks, we update the variables to be the 'ops' variables.  Technically, we should have written these tasks to run twice with a single set of tasks: once with non-ops and the other with ops.  The four variables in question are found in the vars/main.yaml